service/controller/key/key.go

```diff
@@ -260,6 +271,11 @@ func MasterSecurityGroupName(customObject providerv1alpha1.AzureConfig) string {
 	return fmt.Sprintf("%s-%s", ClusterID(customObject), masterSecurityGroupSuffix)
 }
 
+// OSVersion returns the version of the operating system.
+func OSVersion(release releasev1alpha1.Release) (string, error) {
+	return ComponentVersion(release, ContainerLinuxComponentName)
```

Comment:
While I personally do like these for conciseness, we need to have the additional blocks to not swallow steps from stack trace:
```suggestion
	v, err := ComponentVersion(release, ContainerLinuxComponentName)
	if err != nil {
		return "", microerror.Mask(err)
	}
	return v, nil
```